example_test.go

```diff
@@ -109,6 +109,20 @@ func ExampleClient_Incr() {
 	// Output: 1 <nil>
 }
 
+func ExampleClient_BLPop() {
+	if err := client.RPush("queue", "message").Err(); err != nil {
+		panic(err)
+	}
+
+	result, err := client.BLPop(1*time.Second, "queue").Result()
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println(result[0], result[1])
+	// Output: queue, message
```

Comment:
Sorry for stupid mistake; I just didn't found a way to run tests locally
(didn't found contributing process description) and forgot to check travis
build afterward. I've removed comma, so build should be green now.


--
Leonid Shagabutdinov,
+7 905 937 6988

On Tue, Nov 24, 2015 at 7:41 PM, Vladimir Mihailenco <
notifications@github.com> wrote:

> In example_test.go
> <https://github.com/go-redis/redis/pull/197#discussion_r45735817>:
>
> > @@ -109,6 +109,20 @@ func ExampleClient_Incr() {
> >  	// Output: 1 <nil>
> >  }
> >
> > +func ExampleClient_BLPop() {
> > +	if err := client.RPush("queue", "message").Err(); err != nil {
> > +		panic(err)
> > +	}
> > +
> > +	result, err := client.BLPop(1*time.Second, "queue").Result()
> > +	if err != nil {
> > +		panic(err)
> > +	}
> > +
> > +	fmt.Println(result[0], result[1])
> > +	// Output: queue, message
>
> Tests fail with:
>
> got:
>
> queue message
>
> want:
>
> queue, message
>
> Can you please fix it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/go-redis/redis/pull/197/files#r45735817>.
>